CONTRIBUTING.md

```diff
@@ -174,23 +174,29 @@ When submitting the PR remember to label it with the 📖 (:book:) icon.
 
 ## Releases
 
-Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases. Issues in a release milestone have been prioritized and accepted for the release. However, these issues are not committed to the release, unless they are marked as `kind/release-blocking`. Getting them into the release is dependent on someone in the community getting assigned to the issue and completing the work.
-
 - Minor versions CAN be planned and scheduled for each quarter, or sooner if necessary.
   - Each minor version is preceded with one or more planning session.
   - Planning consists of one or more backlog grooming meetings, roadmap amendments,
     and CAEP proposal reviews.
-- Patch versions CAN be planned and scheduled each month for each of the currently supported series (usually N and N-1).
-- Code freeze is in effect 72 hours (3 days) before a release.
+  - Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track work
+    for minor releases. 
+  - Adding an issue to a milestone provides forward visibility on what the next release will be, so, as soon as there
+    is the intent to work on an issue for a target release, contributors are expected to work with maintainers to 
```

Comment:
Are we planning to do this for every single issue that get's implemented? Or is just something we would do if we want to if a contributor wants to set the expectation that it lands in a release?

Or maybe only for bigger issues?

Sounds like a lot of work/toil for us if we do this for every issue that someone starts working on.

Reply:
answered in https://github.com/kubernetes-sigs/cluster-api/pull/6828#discussion_r931286434